### PR TITLE
[MIG] payment_mercadopago: Migration to 16.0

### PR DESCRIPTION
**Pull request by maq-adhoc**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://runbot.adhoc.com.ar/plugberry/mercadopago/pull/45).

**Review by jcadhoc — DISMISSED** [review]
Seems good

**Comment by jcadhoc** [comment]
@roboadhoc r+

File: payment_mercadopago/controllers/main.py [hunk 1]
```diff
@@ -19,21 +19,33 @@ class MercadoPagoController(http.Controller):
...
                  public client key
         :rtype: dict
         """
+        provider_ref = False
+        bin = False
+
         if flow == "token":
-            acquirer_sudo = request.env['payment.token'].browse(rec_id).acquirer_id.sudo()
+            token_sudo = request.env['payment.token'].browse(rec_id).sudo()
+            provider_sudo = token_sudo.provider_id.sudo()
```
Comment (bruno-zanotti) [inline payment_mercadopago/controllers/main.py]: No haría falta el segundo sudo()

File: payment_mercadopago/models/payment_provider.py [hunk 2]
```diff
@@ -55,15 +55,25 @@ class PaymentAcquirer(models.Model):
...
+        ],
+        string='Capture method',
+        default='deferred_capture'
+    )
 
-    @api.onchange('provider')
-    def _onchange_provider(self):
-        if self.provider == 'mercadopago':
+    def get_mercadopago_request(self):
+        return MercadoPagoAPI(self)
```
Comment (bruno-zanotti) [inline payment_mercadopago/models/payment_provider.py]: Me gusta este método, lo haría privado `_get_mercadopago_request(self)`

**Comment by roboadhoc** [comment]
I didn't know about this PR and had to retrieve its information, you may have to re-approve it as I didn't see previous commands.

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/plugberry/mercadopago/pull/45).

**Comment by roboadhoc** [comment]
I'm sorry, @jcadhoc. I'm afraid I can't do that.

**Comment by bruno-zanotti** [comment]
@roboadhoc r+ rebase-ff

**Comment by roboadhoc** [comment]
I'm sorry, @Bruno-Zanotti. I'm afraid I can't do that.

**Comment by nicomacr** [comment]
@roboadhoc rebase-ff

**Comment by roboadhoc** [comment]
I'm sorry, @nicomacr. I'm afraid I can't do that.

**Pull request merged**